### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by jdkandersson**
Fixing a heading issue in the changelog

### CODE REVIEW

File: src/reconcile.py [hunk 1]
```diff
@@ -89,10 +89,10 @@ def _local_and_server(
         The action to execute against the server.
 
     Raises:
-        ReconcilliationError: if the table path or level do not match for the path info and table
-            row.
-        ReconcilliationError: if certain edge cases occur that are not expected, such as
-            table_row.navlink.link for a page on the server.
+        ReconcilliationError:
+            - If the table path or level do not match for the path info and table row
```
Comment (gregory-schiano) [inline src/reconcile.py:93]: Just a small detail but the final "." is missing

**Review by amandahla — APPROVED** [review]
LGTM

File: src/reconcile.py (hunk 1 above)
Comment (jdkandersson) [reply src/reconcile.py]: Thanks, fixed

**Review by arturo-seijas — APPROVED** [review]
LGTM

**Review by nrobinaubertin — APPROVED** [review]

**Pull request merged**